src/compiler/sys.ts

```diff
@@ -221,7 +228,7 @@ namespace ts {
 
             // average async stat takes about 30 microseconds
             // set chunk size to do 30 files in < 1 millisecond
-            function createWatchedFileSet(interval = 2500, chunkSize = 30) {
+            function createPollingWatchedFileSet(interval = 2500, chunkSize = 30) {

```

Comment:
so why do we need this Polling one, why can not we just fall back to fs.watchFile?